src/client/components/storybook/fullscreen.tsx

```diff
@@ -0,0 +1,11 @@
+import { RenderFunction } from '@storybook/react'
+import * as React from 'react'
+
+import * as style from './fullscreen.css'
+
+export function fullscreen(story: RenderFunction) {
+  return <div className={style.fullscreen}>
+    <style>{`body { margin: 0; padding: 0; }`}</style>
```

Comment:
Clever! I'm sure there is a better way to achieve this haha, but this is probably fine for now.

Can you add a comment why this is needed?

Reply:
Done!